### ISSUE TITLE
Add `useBigInt` option to TypeScript definitions

### ISSUE THREAD

**Issue by jhurliman** [sql-js/sql.js#474]
The current copy of https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sql.js/index.d.ts does not have any reference to `useBigInt`, so I am unable to pass that option in to `exec()` statements. Whoever is the maintainer of those TypeScript defs, could you please add this new option?

**Comment by Taytay** [comment]
@lovasoa : I wonder if we should pull those sets into this repo instead? I guess one way would be to rewrite the JS in Typescript so they are guaranteed to be correct. I know some forks have taken that approach. Would you be up for reviewing a PR if I were to do that and make the Typescript compilation part of the build process?

**Comment by lovasoa** [comment]
> Would you be up for reviewing a PR if I were to do that and make the Typescript compilation part of the build process?

Yes ! I am a little bit busy right now, but I do agree we should switch to typescript.

**Comment by rogerfar** [comment]
That would be a great idea, if you need any help with doing some of that I can help in the next few weeks.
